docs/convert.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[convert-to-ecs]]
+== Converting an implementation to ECS
+
+A common schema helps you correlate and use data from various sources. 
+
+Fields for most Elastic modules and solutions (version 7.1 and later) are mapped
```

Comment:
Should be 7.0

```suggestion
Fields for most Elastic modules and solutions (version 7.0 and later) are mapped
```

Reply:
Loving these suggestions :-) @karenzone plz let me know if they're useless to you :-)